internal/app/app.go

```diff
@@ -49,13 +50,16 @@ type DynamoClient interface {
 	Put(ctx context.Context, v interface{}) error
 	Create(ctx context.Context, v interface{}) error
 	DeleteKeys(ctx context.Context, keys []dynamo.Key) error
+	DeleteOne(ctx context.Context, key dynamo.Key) error
```

Comment:
maybe I should have made it `DeleteKeys(ctx context.Context, keys ...dynamo.Key)`